src/App.js

```diff
@@ -1,24 +1,55 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+
+import Header from "./components/Header/Header.js";
+import Footer from "./components/Footer/Footer.js";
+import Main from "./components/Main/Main.js";
+import Form from "./components/Form/Form";
+import Input from "./components/Input/Input";
+import Label from "./components/Label/Label";
+import Textarea from "./components/Textarea/Textarea";
+import EntriesNavigation from "./components/EntriesNavigation/EntriesNavigation";
+import Badge from "./components/Badge/Badge";
+import Tab from "./components/Tab/Tab";
+import EntriesCard from "./components/EntriesCard/EntriesCard";
+import CardDate from "./components/CardDate/CardDate";
+import CardTitle from "./components/CardTitle/CardTitle";
+import Star from "./components/Star/Star";
+import CardText from "./components/CatdText/CardText";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```

Comment:
You could import the children- and grandchildren-components of the main parent components (`Form`, `Section`, `Footer`) inside the parent components themselves so you don't have to import everything in `App.js`.
E.g. the`Form` component will not likely be changed in you journal app so you could build the `Form` component directly in `Form.js` and then just nest it in `App.js`.
Same with the other parent components.

Reply:
Thx, changed the structure now